quantum/keyboard.c

```diff
@@ -669,6 +672,12 @@ void keyboard_task(void) {
     }
 #endif
 
+#ifdef POTENTIOMETER_ENABLE
+    if (potentiometer_task()) {
+        activity_has_occurred = true;
+    }
```

Comment:
This probably needs something which updates `last_input_activity_time()`, so that, e.g., QP displays would also be woken up.

Reply:
Not forgotten, just being lazy while the rest is being decided.